bpf/lib/ipv6.h

```diff
@@ -60,11 +60,13 @@ static __always_inline int ipv6_hdrlen(struct __ctx_buff *ctx, __u8 *nexthdr)
 			if (ctx_load_bytes(ctx, ETH_HLEN + len, &opthdr, sizeof(opthdr)) < 0)
 				return DROP_INVALID;
 
-			nh = opthdr.nexthdr;
 			if (nh == NEXTHDR_AUTH)
 				len += ipv6_authlen(&opthdr);
 			else
 				len += ipv6_optlen(&opthdr);
+
+			nh = opthdr.nexthdr;
+
```

Comment:
small nit: could you remove the extra added newline before the break? Otherwise lgtm, thanks! Please also add a proper commit description, e.g. you could add the PR description also as a commit description so that from git log it is easy to find. Generally, we discourage empty commit descriptions, especially when it comes to bug fixes. Thanks for the fix zcy!

Reply:
adjusted

Reply:
It's still missing the `Fixes:` tag for the commit that introduced the bug. Is that commit unknown?

Reply:
Sorry for my mistake. I added the commit that introduced this bug.

Reply:
Oh, wow! That one is old. Thanks for digging this out!